buildlib/pr/main.yml

```diff
@@ -174,3 +177,17 @@ stages:
         parameters:
             name: gpu
             demands: ucx_gpu -equals yes
+
+  - stage: ucx-py
+    dependsOn: [Static_check]
+    jobs:
+      - template: ucx-py/ucx-py-test.yml
```

Comment:
we should use "python" naming in the context of UCX python bindings (not "ucx-py")
it would be consistent with "java" and "go"

Reply:
But java bindings are named `jucx`: https://github.com/openucx/ucx/tree/master/buildlib/jucx . I can rename that if you prefer, it's just that `ucx-py` is the package name we use currently.

Reply:
I prefer to use this opportunity to rename it to "python", as it becomes part of UCX, also to avoid confusion with the already existing ucx-py project. Re java - moving forward may we will rename it as well, but will be good to start with consistent naming for all bindings.

Reply:
Ok, for now I renamed the directory/files to python in https://github.com/openucx/ucx/pull/7412/commits/41155ea9b23d611691425f3cd9eb95e286d41b6c, but the package will continue to be named `ucx-py`, we will need to discuss how to better address renaming the package.

Reply:
let's also rename lines 180, 183, 188 and other places from "ucx-py" to python , and just keep the package name "ucx-py"